BitwardenShared/UI/Vault/Vault/VaultList/VaultListView.swift

```diff
@@ -67,6 +70,36 @@ private struct SearchableVaultListView: View {
 
     // MARK: Private Properties
 
+    /// A floating action button positioned at the bottom-right corner of the screen.
+    @ViewBuilder private var newLoginFloatingActionButton: some View {
+        VStack {
+            Spacer()
+            HStack {
+                Spacer()
+                Button {
+                    store.send(.addItemPressed)
+                } label: {
+                    Circle()
+                        .fill(Asset.Colors.buttonFilledBackground.swiftUIColor)
+                        .frame(width: 50, height: 50)
+                        .overlay(
+                            Asset.Images.plus.swiftUIImage
+                                .imageStyle(
+                                    .init(
+                                        color: Asset.Colors.buttonFilledForeground.swiftUIColor,
+                                        scaleWithFont: false,
+                                        width: 24,
+                                        height: 24
+                                    )
```

Comment:
🎨 This pressed state is slightly different than our other buttons. Should they match?

| Add an Item | FAB |
| --- | --- |
| <img width="282" alt="Screenshot 2024-10-09 at 1 37 51 PM" src="https://github.com/user-attachments/assets/3c26a400-265d-49ed-b62a-b49815a80b73"> | <img width="84" alt="Screenshot 2024-10-09 at 1 37 58 PM" src="https://github.com/user-attachments/assets/62534cf4-b91e-433f-9726-378172f9d857"> |

I think the FAB is also used on `VaultGroupView`, so maybe making this its own view makes sense.

Reply:
yeah they should match now that i am thinking about it.  Ill make the update.